### PR TITLE
[KEYCLOAK-19833] Fix command line completion on ZSH

### PR DESCRIPTION
**Pull request by andreaTP**
This fix the currently broken command line completion feature on `ZSH`.
The root problem is that having `./` in the generated(by `picocli`) function names breaks the completion script.
This also better conform to `picocli` "canonical" usage.

I tested manually on `ZSH` and `bash`(to avoid regressions) and now everything looks good.

cc. @DGuhr @pedroigor

### CODE REVIEW

**Review by pedroigor — APPROVED** [review]
@andreaTP Finally :) Thanks for this one.

File: quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/command/Main.java [hunk 1]
```diff
@@ -33,7 +33,10 @@
                 "",
                 "Find more information at: https://www.keycloak.org/docs/latest"
         },
-        description = "%nUse this command-line tool to manage your Keycloak cluster.",
+        description = {
+                "Use this command-line tool to manage your Keycloak cluster.",
```
Comment (pedroigor) [inline quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/command/Main.java:37]: @andreaTP Could you please send another PR including a `%n` in the begining of the description. I missed that during my review. It broke the format.

**Comment by andreaTP** [comment]
quick follow up that I took the time to follow up upstream on this issue and it will eventually be fixed in a future release:
https://github.com/remkop/picocli/pull/1477#issuecomment-977543539

cc. @pedroigor @DGuhr
